scripts/linking.py

```diff
@@ -0,0 +1,103 @@
+"""
+Linking based on simple text normalization and exact matching
+"""
+
+import json
+import argparse
+from collections import defaultdict
+
+def normalize(text):
+    normalized = " ".join(sorted(text.lower().split(" ")))
```

Comment:
why is there a `sorted` operation here?  doesnt this break usage for exact match?

Reply:
removed